packages/db/env/run.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 
-PWD=`pwd`/`dirname ${BASH_SOURCE}`
+ENV_DIR=`pwd`/`dirname ${BASH_SOURCE}`
```

Comment:
What is the problem with PWD ? Also, never name things with what there are, no need to put `var` in a variable, we all know it is a variable.